sdk/cosmos/azure-cosmos/azure/cosmos/documents.py

```diff
@@ -308,6 +308,13 @@ class ConnectionPolicy:  # pylint: disable=too-many-instance-attributes
         locations in this list are specified as the names of the azure Cosmos
         locations like, 'West US', 'East US', 'Central India' and so on.
     :vartype PreferredLocations: List[str]
+    :ivar ExcludedLocations:
+        Gets or sets the excluded locations for geo-replicated database
+        accounts. When ExcludedLocations is non-empty, the client will skip this
+        set of locations from the final location evaluation. The locations in
+        this list are specified as the names of the azure Cosmos locations like,
+        'West US', 'East US', 'Central India' and so on.
+    :vartype ExcludedLocations: ~CosmosExcludedLocations
```

Comment:
should this be the same type as the PreferredLocations above (List[str])?

Reply:
Seems like a bug? it should be list of string.